auth-web/src/components/auth/account-freeze/PaymentReview.vue

```diff
@@ -78,10 +86,14 @@ import Steppable from '@/components/auth/common/stepper/Steppable.vue'
 })
 export default class PaymentReview extends Mixins(Steppable) {
   private readonly currentOrganization!: Organization
+  private readonly createAccountPayment!: () => Payment
   private isAcknowledged: boolean = false
 
-  private proceed () {
-    // proceed final step
+  private async proceedToPayment () {
+    const payment:Payment = await this.createAccountPayment()
```

Comment:
can we move this to the view file of this stepper?

do emit('final-step-action') in this function.
and handle that in the Stepper view.(parent)

similar to other steppers.

Reply:
okay..didnt know we have to do..

Reply:
Done.Do you mind having a look?